lib/classes/PluginManager.js

```diff
@@ -41,9 +47,12 @@ class PluginManager {
     }
 
     forEach(options, (value, key) => {
-      if (value.required && (this.options[key] === true || !(this.options[key]))) {
-        const errorMessage = `This command requires the --${key} option. Please pass
-     it through. :)`;
+      if (value.required && (this.cliOptions[key] === true || !(this.cliOptions[key]))) {
+        let requiredThings = `the --${key} option`;
+        if (value.shortcut) {
+          requiredThings += ` / -${value.shortcut} shortcut`;
+        }
+        const errorMessage = `This command requires ${requiredThings}.`;
```

Comment:
how would a shortcut be "required"? I thought a shortcut is just a shortcut for an option, which might be required.